### PR TITLE
Update cocoapods config

### PR DESCRIPTION
**Pull request by SergeyKuryanov**
Bump version in cocoapods config to 1.2.2.
As a bonus dates updated.

### CODE REVIEW

**Comment by SergeyKuryanov** [comment]
To be up to date with current year. Revert?

**Comment by adamyanalunas** [comment]
Yes, please revert them. The only change that matters here is data for updating the version. I can’t even remember if that version string is the only thing that needs changing.

**Comment by SergeyKuryanov** [comment]
@adamyanalunas reverted.

**Comment by SergeyKuryanov** [comment]
@adamyanalunas added.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+### Version 1.2.2
+
+* Fixed issue when builded as static library using cocoapods
```
Comment (adamyanalunas) [inline CHANGELOG.md]: What about

> Fixed issue preventing building as static library using CocoaPods

**Comment by SergeyKuryanov** [comment]
@adamyanalunas done.

**Pull request merged**